automated/linux/ota-rollback/verify-rollback.sh

```diff
@@ -85,6 +86,25 @@ else
     report_skip "bootfirmware_version_after_rollback"
     report_skip "fiovb_is_secondary_boot_after_rollback"
 fi
+
+if [ "${TYPE}" = "app" ]; then
+    echo "Checking for broken app"
+    runtime="5 minute"
+    endtime=$(date -ud "$runtime" +%s)
+
+    while [ "$(date -u +%s)" -le "$endtime" ] && [ ! "${FOUND}" -eq 1 ]; do
+        journalctl --no-pager -u aktualizr-lite
+        echo "Sleep for 1 min"
+        sleep 60
+        if (journalctl --no-pager -u aktualizr-lite | grep "Finalization has failed; reason: apps start failure"); then
```

Comment:
maybe there is a way to determine the state by MESSAGE and RESULT variables from aklite?